src/resources/postcss/tickets-commerce/admin/orders/_svgs.pcss

```
@@ -0,0 +1,11 @@
+/**
+ * Event Tickets - Orders SVGs
+ *
+ * @since TBD
+ */
+
+@svg-load tec-order-management-trash-icon-red url('../../../../images/admin/trash.svg') {
+	path {
+		fill: #B32D2E;
```

e always use lowercase for hex colors
```suggestion
		fill: #b32d2e;
```